src/uwtools/cli.py

```diff
@@ -74,7 +74,6 @@ def _add_subparser_config(subparsers: Subparsers) -> ModeChecks:
     return {
         STR.compare: _add_subparser_config_compare(subparsers),
         STR.realize: _add_subparser_config_realize(subparsers),
-        STR.translate: _add_subparser_config_translate(subparsers),
```

Comment:
Various mechanics to move `uw config translate` to `uw template translate` follow in this module.